src/applications/gi/components/heading/Breadcrumbs.jsx

```diff
@@ -1,31 +1,34 @@
 import { Link, browserHistory } from 'react-router';
 import PropTypes from 'prop-types';
 import React from 'react';
+import Breadcrumbs from '@department-of-veterans-affairs/formation/Breadcrumbs';
 
-class Breadcrumbs extends React.Component {
+class GiBillBreadcrumbs extends React.Component {
   render() {
     const {
       pathname,
       query: { version },
+      search,
     } = this.props.location;
+    const root = { pathname: '/', query: version ? { version } : {} };
+    const facilityCode = this.props.facilityCode;
 
     const crumbs = [
       <a href="/" key="home">
         Home
       </a>,
       <a href="/education/" key="education">
-        Education Benefits
-      </a>,
-      <a href="/education/gi-bill/" key="gi-bill">
-        GI Bill
+        Education and Training
       </a>,
+      <Link to={root} key="main">
+        GI Bill Comparison Tool
+      </Link>,
     ];
 
-    if (pathname.match(/search|profile/)) {
-      const root = { pathname: '/', query: version ? { version } : {} };
+    if (pathname.match(/search/)) {
```

Comment:
Per our discussion, I reviewed search result breadcrumb and functionality with @mnorthuis for correct behavior. We do want search results included, and to remember that search when possible. If users copy and paste a profile URL, it will drop out the search results and just have 4 breadcrumb links.